src/NexusMods.App.UI/Controls/Settings/SettingEntries/SettingInteractionControls/Toggle/SettingToggleControl.axaml

```diff
@@ -8,7 +8,7 @@
     mc:Ignorable="d"
     x:Class="NexusMods.App.UI.Controls.Settings.SettingEntries.SettingToggleControl">
 
-    <ToggleSwitch HorizontalAlignment="Center" x:Name="ToggleSwitch">
+    <ToggleSwitch Classes="Compact" HorizontalAlignment="Center" x:Name="ToggleSwitch" ClipToBounds="False">
```

Comment:
Here as well I would not expect ClipToBounds to be required since it's already set in the style